rpm/libphonenumber.spec

```diff
@@ -51,6 +53,9 @@ Contains files needed to development with %{name}.
 %prep
 %autosetup -p1 -n %{name}-%{version}/%{name}
 
+sed -r -i 's/\b(CMAKE_CXX_STANDARD[[:blank:]]+)11\b/\117/' \
+    cpp/CMakeLists.txt tools/cpp/CMakeLists.txt
```

Comment:
Curious why this. So the new version requires newer c++ standard but how is the upstream not having such enabled on the build files?

Could have some comment on why here?